Automattic/jetpack
# Subscribe block: add "show login" toggle

Adds a toggle to "show login" (better copy needed) under subscribe block.

Then in the rest of the features using subscribe block we can simply add attribute to show login e.g. at paywall and subscribe modal.

<img width="1064" alt="Screenshot 2023-10-25 at 18 27 21" src="https://github.com/Automattic/jetpack/assets/87168/96b372d8-7a28-4a35-bbfa-eebd22b31262">


## Proposed changes:

* Add "show login" toggle to subscribe block settings

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add subscribe block without this branch; then switch to branch and ensure the block works and doesn't invalidate.
* Enable "show login" and ensure editor and frontend look good.
* Make sure the link still works.
